src/features/class-hello.php

```diff
@@ -0,0 +1,94 @@
+<?php
```

Comment:
Let's move this into the features README as an example of a feature class so it's not a file that needs to be deleted after initializing the plugin